src/main/java/seedu/address/logic/commands/undocommand/UndoCommand.java

```diff
@@ -24,8 +24,7 @@
     public CommandResult execute(Model model) throws CommandException {
         try {
             model.undo();
-            return new CommandResult(String.format(MESSAGE_SUCCESS),
-                    CommandType.P); //Debug: Check PrefixType
+            return new CommandResult(String.format(MESSAGE_SUCCESS), CommandType.H);
```

Comment:
For the undo command, do you want the GUI to display the respective EntityList instead? to show what  has been undone? :)

Reply:
I was thinking of that as well, but I'm not sure which of the 2 options to go with. Right now, i'm more inclined towards displaying History, because the user can then see how his/her current state has changed as compared to before the undo command was executed. What do you think?

Reply:
I think showing the affected EntityList would be better cuz when we ctrl+z it's not like we want to see which actions we can undo next